corehq/apps/reports/standard/cases/case_list_explorer.py

```diff
@@ -4,10 +4,31 @@
 
 from corehq.apps.es.case_search import CaseSearchES
 from corehq.apps.reports.standard.cases.basic import CaseListReport
+from corehq.apps.reports.standard.cases.filters import (
+    XpathCaseSearchFilter,
+)
 
 
 class CaseListExplorer(CaseListReport):
     name = _('Case List Explorer')
     slug = 'case_list_explorer'
     search_class = CaseSearchES
 
+    fields = [
+        'corehq.apps.reports.filters.case_list.CaseListFilter',
+        'corehq.apps.reports.filters.select.CaseTypeFilter',
+        'corehq.apps.reports.filters.select.SelectOpenCloseFilter',
+        XpathCaseSearchFilter,
```

Comment:
why the mix of string module paths and actual classes?

Reply:
I agree, this is weird. I'd imported these two classes into this file. Will update.